src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -904,9 +904,13 @@ MulticopterPositionControl::cross_sphere_line(const math::Vector<3> &sphere_c, f
 
 void MulticopterPositionControl::control_auto(float dt)
 {
-	if (!_mode_auto) {
+	if (!_mode_auto || _vehicle_status.in_transition_mode || !_vehicle_status.is_rotary_wing) {
```

Comment:
Please add a comment for why this condition is how it is. A reader has no clue what's going on here.